rest/src/test/resources/db/migration/V202010042030__baseline.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE ATTRIBUTE
```

Comment:
Maybe those scripts could be used from the main application. So you cannot forget to copy the migrations